aspnet/fundamentals/localization.rst

```diff
@@ -204,6 +204,9 @@ CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
```

Comment:
Didn't see @danroth27's comment when I made this comment but I think he and I meant the same thing. The reference here is out of date and `ASPNET_CULTURE` should be changed to `.AspNetCore.Culture`.